erikiado/jp2_online
# Task/asignacion beca

#### What's this PR do?
- Adds the functionality to assign scholarships.
- Modifies the model of Beca to store percentage instead of raw amount.
- Deletes the Dashboard section for an admin.

#### Where should the reviewer start?
There's a lot of changed files, but note that many of them are just changes in tests or minor refactoring.

The most important thing is:
becas/models
becas/views

templates/becas
js/becas

#### How should this be manually tested?
besides running the tests, check that you can assign scholarships to studies that are approved. to check that these are saved, you can check the info of students in the search tab.

#### Any background context you want to provide?
Ive thought about some details such as checking if the family has members before giving the scholarship (right now that would not break anything but the view would be useless), but I think we should validate that before approving the study, or before even submitting it for revision.

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
